src/Bindings/PluginLua.cpp

```diff
@@ -26,6 +26,7 @@ extern "C"
 #undef TOLUA_TEMPLATE_BIND
 #include "tolua++/include/tolua++.h"
 
+const std::string LUA_FILE_SUFFIX = ".lua";
```

Comment:
Move it into the loader function and use `static const AString` as the datatype.